### DISCUSSION TITLE
Set alpha to 1 if larger than 0

### DISCUSSION THREAD

**Discussion by bhenriksson031** [cables-gl/cables_docs#234] [Answers & Questions]
I got a semitransparent alpha and I want to mask out those areas completely. So I want to do this:  
if(alpha>0) alpha=1;

**Comment by pandrr** [comment]
whats the context ?  shader ? texture ? normal math ops ?

**Reply by bhenriksson031** [reply]
Texture. I managed to do it in a custom shader after render on a plane but would need to do it on a image I pass to various Draw image nodes.
